lib/phobos_db_checkpoint.rb

```diff
@@ -20,48 +20,69 @@ module PhobosDBCheckpoint
 
   class << self
     attr_reader :db_config
-    attr_accessor :db_config_path, :db_dir, :migration_path
+    attr_accessor :db_config_path, :db_dir, :migration_path, :verbose
```

Comment:
I think it makes sense to just use debug-level logging instead of `verbose` with `puts` option

Reply:
I agree, But right now we don't have any logger for PhobosDbCheckpoint itself. So either we need to add one or piggyback on Phobos logger, which might be better. Either way, I think logging is outside of this PR so I just removed the output for now since it wasn't really adding that much extra anyway...